intranet/templates/dashboard/dashboard.html

```diff
@@ -206,15 +207,30 @@ <h3>
             </div>
         {% endfor %}
         {% if not request.user.is_restricted %}
-        {% if start_num == 0 and view_announcements_url != 'announcements_archive' %}
+        {% if page_num == 1 and view_announcements_url != 'announcements_archive' %}
             <a href="{% url 'announcements_archive' %}" class="button" style="float:left"><i class="fas fa-archive" style="width: 13px"></i> View Archive</a>
         {% endif %}
 
-        {% if start_num > 0 %}
-            <a href="{% url view_announcements_url %}?start={{ prev_page }}{{ paginate_link_suffix }}" class="button" style="float:left">&larr; Newer Posts</a>
+        {% if prev_page > 0 %}
+            <a href="{% url view_announcements_url %}?page={{ prev_page }}{{ paginate_link_suffix }}" class="button" style="float:left">&larr; Newer Posts</a>
         {% endif %}
         {% if more_items %}
```

Comment:
You've hidden the pagination UI when there aren't `more_items` - this does solve the initial issue by hiding the UI when announcements only take up a single page. However, it seems to introduce another issue: when there are multiple pages of announcements, the UI is hidden on the last page (since there aren't any more items). I assume this is an accidental bug - or was it intentional?